vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_havana_errors_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 describe EmsRefresh::Refreshers::OpenstackRefresher do
   before(:each) do
+    EmsRefresh.debug_failures = false
+
```

Comment:
Oh I see...I didn't catch that these were testing the actual error-handling situation.